### ISSUE TITLE
Check if use of Ubuntu 22.04 is already possible

### ISSUE THREAD

**Issue by berendt** [osism/k8s-capi-images#162]

**Comment by berendt** [comment]
Not yet available for qemu.

```
qemu-centos-7.json
qemu-flatcar.json
qemu-rhel-8.json
qemu-rockylinux-8.json
qemu-ubuntu-1804.json
qemu-ubuntu-2004-efi.json
qemu-ubuntu-2004.json
```

**Comment by berendt** [comment]
Fixed with e351b4a0709b845ad109b2a09bd5d613c9c06b81 and 3363ec062b67b1e86add7203f40c7716fcb3c637.
